torch/_inductor/cudagraph_trees.py

```diff
@@ -2008,7 +2050,13 @@ def exceed_rerecord_limit(
             > torch._inductor.config.triton.cudagraph_unexpected_rerecord_limit
         )
 
-    def _run(self, new_inputs: List[InputType], function_id: FunctionID) -> OutputType:
+    def _run(
+        self,
+        new_inputs: List[InputType],
+        function_id: FunctionID,
+        mode: CompilationMode,
```

Comment:
Mode is mapped by `self.id_to_mode`, shouldn't have to pass in explicitly. can we do something similar with function_id as well ?